RapidReact/src/main/java/frc/robot/Robot.java

```diff
@@ -338,41 +344,32 @@ private void configureButtonBindings() {
         )
         .whenReleased(() -> driverClimbEnabledPressed = false);
 
-      buttons.elevatorExtend.whenPressed(climberSubsystem::elevatorExtend);
+      buttons.elevatorExtend.whenPressed(climberSubsystem::manualElevatorExtend);
       buttons.elevatorExtend.whenReleased(climberSubsystem::elevatorStop);
 
-      buttons.elevatorRetract.whenPressed(climberSubsystem::elevatorRetract);
+      buttons.elevatorRetract.whenPressed(climberSubsystem::manualElevatorRetract);
       buttons.elevatorRetract.whenReleased(climberSubsystem::elevatorStop);
-
-      buttons.climbAuto.whenPressed(
-        () -> {
-          climberSubsystem.climbAuto();
-          rgbSubsystem.autoClimb();
-        }
-      );
+      
+      buttons.climbAuto.whenPressed(climberSubsystem::autoClimb);
+      buttons.climbAuto.whenReleased(climberSubsystem::autoClimbReleased);
+      buttons.resetClimbStuff.whenPressed(climberSubsystem::resetClimbStuff);
```

Comment:
We need to make sure these only work after the climber state is enabled (by pressing both PS4 buttons). We don't want the operator to accidentally hit the trackpad button and trigger an auto climb.